maidroid_core/cores/ocr.lua

```diff
@@ -327,19 +325,42 @@ local maidroid_instruction_set = {
 			under = under,
 			type = "node"
 		}
-		local newitem, succ = stack:get_definition().on_place(stack, obj, pt)
+		local stack_def = stack:get_definition()
+		local newitem, succ = stack_def.on_place(stack, thread.droid.object, pt)
```

Comment:
Accordingly to [this](https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L3850), placer can be **any** objectref or **nil**. But with nil I get a crash from builtin. :P